include/svnlook.php

```diff
@@ -876,6 +876,11 @@ function listReadmeContents($path, $rev = 0, $peg = '') {
 
 		$file = "README.md";
 
+		if ($this->isFile($path.$file) != True)
```

Comment:
Please update the comment above as well and any other instances referring to an uppercase `README.md`